Docs/Configuration.tex

```diff
@@ -821,10 +821,9 @@ \subsection{Introduction}\label{acpiintro}
 \end{itemize}
 
 Applying the changes globally resolves the problems of incorrect operating system
-detection, which is not possible before the operating system boots according to
-the ACPI specification, operating system chainloading, and harder ACPI debugging.
-For this reason it may be required to carefully use \texttt{\_OSI} method when
-writing the changes.
+detection (consistent with the ACPI specification, not possible before the operating
+system boots), operating system chainloading, and difficult ACPI debugging. Hence,
+carefully using the \texttt{\_OSI} method may be required when writing changes.
```

Comment:
How about `Hence, more attention may be required when writing changes to \texttt{\_OSI}.`?